willprice/python-omxplayer-wrapper
# Now also recognizing named pipes as input for player

## Description
I added a check to detect if the given filename is a named pipe. Streaming from named pipes works with omxplayer on console so it should also with this wrapper.

* **Status**: *READY*

## Related Issues
https://github.com/willprice/python-omxplayer-wrapper/issues/59

## Steps to Test or Reproduce

```sh
mkfifo fifo.mjpg
change "Hello World" example to load fifo.mjpg
omxplayer-wrapper should not throw a FileNotFoundException
```
